app/clinic/controllers/patientCommonController.js

```diff
@@ -0,0 +1,90 @@
+(function () {
```

Comment:
Duplicating the controller fixes the issue with ```patientCommonController``` not being loaded, but now we have to maintain both this one [and the one in registration](https://github.com/esaude/esaude-emr-poc/blob/master/app/registration/controllers/patientCommonController.js). The amount of work for this functionality is doubled. 

I think the delete button can be easily transformed into a directive, I'm available if you want to work together.

Reply:
@ynurmahomed  Lets do it